scripts/update

```diff
@@ -23,7 +23,7 @@ then
             docker-compose up -d database
             ./scripts/migrate
             
-            docker-compose build
+            docker-compose build database franklin catalogs-aviris activator-aviris-l2 shellcheck black dev
```

Comment:
I see that this is to avoid the build of the `cog-clip` container but I don't understand why.

Reply:
Because update is called before the app assembly jar is built, and we can build it only after running tests; so we can't prebuild its container.

Reply:
Ah, that makes sense. I'd consider removing the `docker-compose build` invocation from `update` all together then. I don't think we need any of these container images built outside of CI.

Reply:
Removed the docker-compose build command.